packages/osd-config/src/env.test.ts

```diff
@@ -59,10 +59,12 @@ test('correctly creates default environment in dev mode.', () => {
     getEnvOptions({
       configs: ['/test/cwd/config/opensearch_dashboards.yml'],
       isDevClusterMaster: true,
+      isDevClusterManager: false,
     })
   );
 
   expect(defaultEnv).toMatchSnapshot('env properties');
+  expect(defaultEnv.isDevClusterManager).toBeTruthy();
```

Comment:
line 62 says false but this is expected to be truthy?

Reply:
> line 62 says false but this is expected to be truthy this is because `isDevClusterMaster` is true then we expect `isDevClusterManager` to be true right? That's great. Can we have a test for the invert. Like `isDevClusterManager` is set to true. What's the expected behavior?

Updated